modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase

```diff
@@ -275,6 +304,40 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test SaveCustomObjectEntityTypeAsTemplate {
+		task ("Given a custom object is published") {
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "student",
+				en_US_plural_label = "students",
+				name = "Student",
+				requiredStringFieldName = "name");
+		}
+
+		task ("And Given several attribute codes are selected in export wizard") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoExport();
+		}
+
```

Comment:
And the failure of the creator field is LPS-189722, you can look at it, and modify your case

Reply:
I added the ignore= true for this test until to resolve LPS-189722